internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -230,6 +231,26 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"serverless_compute": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"custom_subnet_id": {
```

Comment:
This could be shortened to 
```suggestion
						"subnet_id": {
```